doc/source/suse-ecp/ose-localhost.rst

```diff
@@ -69,6 +69,7 @@ Optionally, `localhost` can be preinstalled with the following software:
   * python3-jmespath
   * python3-openstacksdk
   * python3-netaddr
+  * python3-ara
```

Comment:
I don't think we have `python3-ara` in regular Leap 15 repos. AFAIK if you want ara, you need to install it into virtualenv

Reply:
ah, that may be true since I'm using tumbleweed. Let's dismiss this PR then.